tests/csrf_tests/tests.py

```diff
@@ -185,10 +185,21 @@ def test_csrf_token_in_header(self):
         Check that we can pass in the token in a header instead of in the form
         """
         req = self._get_POST_csrf_cookie_request()
-        req.META['HTTP_X_CSRFTOKEN'] = self._csrf_id
+        req.META[settings.CSRF_HEADER_NAME] = self._csrf_id
```

Comment:
Since the default value is keept in settings I thought it would be reasonable to use setting here, moreover I can imagine that in future we change that default name and we should be able to do it in one place, and that change should be transparent for the tests.